app/commands/v2/patch_link_set.rb

```diff
@@ -133,7 +117,7 @@ def schema_validator
 
       def schema_name
         @schema_name ||= Queries::GetLatest.(
-          ContentItem.where(content_id: payload[:content_id])
+          Edition.where(documents: { content_id: content_id }).joins(:document)
```

Comment:
perhaps there's a case for defining this as a scope on `Edition`? seems to know more about the inner workings of the model than you might expect a command to.